### PR TITLE
[IDB IGNORE] Bay clicking nosies 2: The clickening

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Adds new sounds:

Beakers now have a splash, pickup, drop, and transfer sound
~~SMESes, APCS, computers, and other electronics make noise~~ this was removed
Health and mining scanners make a noise when they scan
Guns now have pick up noises
Port's azarak's port of button sprites

I dont remember exactly what i did, but its mostly ported stuff from nebula, bay, bee, tg, and SS14

ports most of https://github.com/hrzntal/horizon/pull/536
ports https://github.com/tgstation/tgstation/pull/54324
ports https://github.com/Baystation12/Baystation12/pull/27256
https://github.com/NebulaSS13/Nebula/commit/72ea520f7682cb1b4d7bfc9b6bf87d1803acaa8c
https://github.com/NebulaSS13/Nebula/pull/1403
https://github.com/BeeStation/BeeStation-Hornet/pull/29
~~https://github.com/space-wizards/space-station-14/pull/4552~~
Beaker sounds are from CEV Eris, however theres no pr so its only this commit https://github.com/discordia-space/CEV-Eris/commit/3dd379a8c41e47bdb03f7c29e4afaf315f35015c
Gun pickups were taken from desert rose, again theres no PR https://github.com/judgex/desertrose/commit/704aa5e9e3b86d6e1dbf64a29ba976f7ef53a4ab
anything else i do not remember adding

## Why It's Good For The Game

sound design good

## Changelog
:cl: Azarak, Qustinnus, comma, mattroks101, metalgearsloth, tichys
add: A shit ton of clicky noises
add: Guns now have pickup noises
soundadd: more ambience
imageadd: some buttons got new sprotses
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
Regarding air alarm emissives. I made a mistake in my air alarm PR that made the air alarms have a missing icon state when they were broken, and the overlays wouldn't respect whether the air alarm is powered or broken. https://github.com/hrzntal/horizon/pull/551 Fixes those issues.

**Comment by rye-rice** [comment]
removing apc, smes, and computer amibent sounds beause byond just does not play well with ambient sounds

**Comment by Kapu1178** [comment]
Credit authors in CL.

**Review by Kapu1178 — DISMISSED** [review]
Please credit all contributors from the original PRs in your changelog.

**Comment by rye-rice** [comment]
m

**Review by triplezeta — APPROVED** [review]
minecraftKepi
im not sure why my review was requested but the AAC resprite is nice

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -450,6 +452,8 @@
 	item_state = "glowstick"
 	grind_results = list(/datum/reagent/phenol = 15, /datum/reagent/hydrogen = 10, /datum/reagent/oxygen = 5) //Meth-in-a-stick
 	var/fuel = 0
+	toggle_on_sound = 'sound/effects/glowstick.ogg'
+	toggle_off_sound = 'sound/effects/glowstick.ogg'
```
Comment (MarkSuckerberg) [inline code/game/objects/items/devices/flashlight.dm:456]: how are you going to toggle a glowstick off

**Review by MarkSuckerberg — APPROVED** [review]
Seems good codewise, TMing

File: code/game/objects/items/devices/flashlight.dm (hunk 1 above)
Comment (Ms-Mee) [reply code/game/objects/items/devices/flashlight.dm:456]: Eat it 😋

**Review by ZephyrTFA — APPROVED** [review]
This is an approving review for code quality and standards, good work

**Comment by rye-rice** [comment]
how do you set dnm

**Comment by rye-rice** [comment]
Alright so, even after removing the ambient sounds, it has a tendancy of not removing punch/hit overlays and making supply pods lag a lot. going to need help with that

**Comment by rye-rice** [comment]
This should be ready for TMing again, i swear if the same issues happen again i will scream

**Comment by rye-rice** [comment]
@Kapu1178

**Comment by ZephyrTFA** [comment]
Is this still causing ambience issues?

**Pull request merged**